src/services/submissionService.ts

```diff
@@ -175,17 +176,38 @@ Promise<Result<gqlTypes.Submission>> => {
         const newSubmissionId = submissionRef.id
         const newSubmission = mapGqlEntityToDbEntity(submissionInput, newSubmissionId)
         
-        await submissionRef.set(newSubmission)
+        const batch = dbInstance.batch()
+        
+        batch.set(submissionRef, newSubmission)
+
+        const audit: dbSchema.AuditLog = {
+            actionType: gqlTypes.ActionType.Create,
+            objectType: gqlTypes.ObjectType.Submission,
+            schemaVersion: gqlTypes.SchemaVersion.V1,
+            updatedBy: 'client',
```

Comment:
In this case this is updated by the user, so not sure if I can just hardcode this or not?